README.md

```diff
@@ -1,5 +1,125 @@
 # Kafka Histo Streamer
 
-Consumes event data from kafka and re-streams it as histogram data
+This is a prototype to test and demonstrate Mantid being capable of live processing event data into histogram data, streaming both via Kafka. Subscribes to a topic with event data and publishes it to another topic as histogram data.
 
+## Requirements
 
+Requires Mantid (currently has to be `nightly`).
+
+Virtualenv Python package recommended and used below (`$ pip install virtualenv`).
+
+Running a local demo additionally requires [Docker](https://www.docker.com/) and [ISIS Nexus Streamer](https://github.com/ScreamingUdder/isis_nexus_streamer_for_mantid/).
+
+Already includes the `hs00` schema, and generated code, from the DMSC [Schema Repository](https://github.com/ess-dmsc/streaming-data-types).
+
+
+## Setup
+
+Ensure [Mantid](https://www.mantidproject.org/Main_Page) is installed.
+
+Clone this repository:
+
+```
+$ git clone https://github.com/DMSC-Instrument-Data/kafka_histo_streamer
+```
+
+
+You will want to edit `run.py` and `postprocess.py` to configure `kafka_server` and `kafka_topic` as needed. If you want to run a local demo, we will revisit what these should be set to below.
+
+## Execution
+
+The Histo Streamer must be executed using `mantidpython`. There are currently no arguments:
```

Comment:
And mantidpython will be found in the bin directory of the Mantid installation i.e. /opt/Mantid/nightly... or whatever the path is.